.github/workflows/pr-operator.yml

```diff
@@ -145,11 +145,8 @@ jobs:
           # Controller-gen
           make controller-gen
 
-          # Kustomize
-          make kustomize
-
```

Comment:
I removed this because CI was failing. `make bundle` also runs `kustomize` so was getting "file exists" error. felt like the best place to fix this issue but happy to revert/change